docs/14_0_0/gettingstarted/whatsnew.md

```diff
@@ -13,6 +13,9 @@ Look into [migration guide](https://primefaces.github.io/primefaces/14_0_0/#/../
     * OS settings for `prefers-reduced-motion: reduce` is now respected and PF disables all animations.
     * Minimum inline Ajax load animation duration via `PrimeFaces.ajax.minLoadAnimation`.
 
+* Chart
+   * A new more flexible version of Chart.js can be used `<p:chart>` allowing raw JSON or XDEV Java Model
+   
```

Comment:
should we mention that the current impl is deprecated and will be remov in later version?